src/configs/settings.ts

```diff
@@ -870,6 +870,45 @@ export const indicatorThresholdsIRS: IndicatorThresholds = {
 };
 /** END IRS Reporting configs */
 
+/** Namibia IRS Reporting configs */
+export const indicatorThresholdsIRSNamibia: IndicatorThresholds = {
+  GREEN_THRESHOLD: {
+    color: '#4C9A2A',
+    name: 'Green',
+    value: 1,
+  },
+  LIGHT_GREEN_THRESHOLD: {
+    color: '#76BA1B',
+    name: 'Light Green',
+    value: 0.85,
+  },
+  RED_THRESHOLD: {
+    color: '#FF4136',
+    name: 'Red',
+    orEquals: true,
+    value: 0.7,
+  },
+  YELLOW_THRESHOLD: {
+    color: '#FFDC00',
+    name: 'Yellow',
+    value: 0.8,
+  },
+};
```

Comment:
Can we pass the `name` strings in from the `lang.ts` file so we are sure they are translated?

Reply:
I've added these to a ticket requesting TH translations for new display strings.